### ISSUE TITLE
[Feature request] Export image of map

### ISSUE THREAD

**Issue by FakenameBill** [kahomayo/monolith-renderer#1]
TL;DR: Allow users to download a .png image of a region of the world bounded by user-supplied X and Z coordinates (preferably without grid lines)

For a while now I've been interested in finding a tool to make top-down visualizations of Alpha worlds showing the division between land and water similar to how AMIDST shows biomes in more recent versions of Minecraft. Recently, AMIDST implemented support for the Alpha 1.2 - Beta 1.7 biome system, including using terrain noise to find the locations of oceans and lakes, but there is still no support for pre-1.2 Alpha worlds. 

While it's quite easy to take a screenshot of the monolith renderer's output, I'm interested in making an image of a very specific region of the world. The goal is to have several images of the same seed's biome/continent maps throughout many different versions of Minecraft in order to compare and contrast changes over time. In order to do so properly, the images must line up exactly. In AMIDST, this is done easily using the "export biomes as image" feature, in which the user supplies specific X and Z coordinates of the region to be exported. A feature like this in the monolith-renderer would allow such an image to be created for Alpha worlds as well since AMIDST doesn't support them. 

If this is out of scope, a description of where in the code the image formatting is done would be greatly appreciated. I have a programming background, but reading and modifying other people's code is intimidating af lmao

**Comment by FakenameBill** [comment]
Ope, just discovered your amidst-infdev tool, which is exactly what I'm looking for
